### PR TITLE
codeine-216 use jdbc connection pool

### PR DESCRIPTION
**Pull request by ezraroi**
This change is [Reviewable](https://reviewable.io/reviews/codeine-cd/codeine/217)

### CODE REVIEW

File: src/common/codeine/db/mysql/DbUtils.java [hunk 1]
```diff
@@ -1,187 +1,254 @@
...
+
+    @Inject
+    private MysqlHostSelector hostSelector;
+
+
+    public DbUtils() {
+        super();
+        genericObjectPoolConfig.setMaxTotal(2);
+        genericObjectPoolConfig.setMinIdle(1);
+        genericObjectPoolConfig.setMaxIdle(2);try {
```
Comment (lchayoun) [inline src/common/codeine/db/mysql/DbUtils.java]: new line before the try

File: src/common/codeine/db/mysql/DbUtils.java [hunk 2]
```diff
@@ -1,187 +1,254 @@
...
+    private DBConnection dbConnection = new DBConnection();
+    private Map<String, String> connectionsPool = Maps.newConcurrentMap();
+
+    @Inject
+    private MysqlHostSelector hostSelector;
+
+
+    public DbUtils() {
+        super();
+        genericObjectPoolConfig.setMaxTotal(2);
```
Comment (lchayoun) [inline src/common/codeine/db/mysql/DbUtils.java]: is there a way to get the numbers from configuration?

Comment (ezraroi) [reply src/common/codeine/db/mysql/DbUtils.java]: no

Comment (ezraroi) [reply src/common/codeine/db/mysql/DbUtils.java]: fixed

**Pull request merged**